src/renderer/components/+preferences/application.tsx

```diff
@@ -93,13 +97,47 @@ export const Application = observer(() => {
 
       <hr/>
 
+      <section id="extensionRegitryUrl">
+        <SubTitle title="Extension Install Registry" />
+        <Select
+          options={Object.values(ExtensionRegistryLocation)}
+          value={userStore.extensionRegistryUrl.location}
+          onChange={action(({ value }) => {
+            userStore.extensionRegistryUrl.location = value;
+
+            if (userStore.extensionRegistryUrl.location === ExtensionRegistryLocation.CUSTOM) {
+              userStore.extensionRegistryUrl.customUrl = "";
+            }
+          })}
+          themeName="lens"
+        />
+        <p className="mt-4 mb-5 leading-relaxed">
+          This setting is to change the registry URL for installing extensions by name.{" "}
+          If you are unable to access the default registry ({defaultExtensionRegistryUrl}) {" "}
+          you should change this to either the registry in you{" "}
```

Comment:
[typo] you => your

Maybe rephrase it `you can change the registry URL in your .npmrc or add a custom URL in the input below`. 

Or something like that.